halo/app/monitor.go

```diff
@@ -167,3 +184,44 @@ func dirSize(path string) (int64, error) {
 
 	return size, nil
 }
+
+// startMonitoringAPI registers metrics and health endpoints.
+func startMonitoringAPI(ctx context.Context, cfg *cmtcfg.Config, cmtNode *node.Node, syncAbort chan error, status *readinessStatus) {
```

Comment:
lets stick to existing naming, rename to `asyncAbort`

Reply:
Yep, sync is a type. Will change the type 👍